meshtastic/slog/slog.py

```
@@ -158,39 +224,77 @@ def __init__(
-        self.raw_file: Optional[
-            io.TextIOWrapper
-        ] = open(  # pylint: disable=consider-using-with
-            os.path.join(dir_path, "raw.txt"), "w", encoding="utf8"
-        )
+        self._raw_file_lock = threading.Lock()
+        self.raw_file: io.TextIOWrapper | None = None
 
         # We need a closure here because the subscription API is very strict about exact arg matching
-        def listen_glue(line, interface):  # pylint: disable=unused-argument
+        def listen_glue(
+            line, interface
```

Ruff (0.15.1)

[warning] Unused function argument: `interface`

(ARG001)